classes/cluster/mk22_lab_advanced/init.yml

```diff
@@ -49,3 +49,7 @@ parameters:
     stacklight_monitor_node01_address: 172.16.10.107
     stacklight_monitor_node02_address: 172.16.10.108
     stacklight_monitor_node03_address: 172.16.10.109
+
+    stacklight_telemetry_node01_address: 172.16.10.107
```

Comment:
you should use ${_param:stacklight_monitor_node01_address} instead to avoid duplication